low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -197,6 +199,29 @@ SYSTEMDWATCHDOG:
    smart_test_interval: 999999999
    run_smart_on_start: 'False'
 
+SERVICEMONITOR:
+   monitor: true
+   threaded: true
+   thread_sleep: 1
+   polling_frequency: 15
```

Comment:
Not a strong reason, but it should depend on `threshold inactive time` allowed for any service. according to the current allowed time of 60, I thought of 2 iterations within that period to reduce buffer time to 30s.

Reply:
for eg; 
If a service fails at time 1
then in the next iteration of 60 seconds time elapsed is 59 so it won't raise an alert
then in another iteration of 60 seconds, we would receive an alert and the time elapsed will be 119 seconds till then

by using polling time of 30s, this will be reduced to 89s

Reply:
threshold inactive time is max time required by listed 3rd party services to restart. 

I said ` it should depend on threshold inactive time` 
currently I kept temp value of  threshold_inactive_time or max_wait_time = 60s, and explained accordingly.

Reply:
sure.. 👍